### DISCUSSION TITLE
Github not opening in my pc

### DISCUSSION THREAD

**Discussion by Anurag2782** [community/community#37965] [Other Feature Feedback, Questions, & Ideas]
![16673001891494873284945472291861](https://user-images.githubusercontent.com/116972643/199218287-e5c2f312-30aa-4d40-bae0-01594ee7d965.jpg)
I have been trying to open github in my pc using different browser but it is not opening and showing "this site is not secure " and "this site has no certificate ". I am trying different things like clear browsing history and cookies and changing dns server etc in different browsers also but it is working when I use VPN in my pc and in my mobile phone it is working.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Anurag2782! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Given that it is working with your VPN and mobile phone this is likely not something the community can help with and may be an internet provider problem.
